src/components/layers-panel/arcgis-import-panel/arcgis-import-panel.tsx

```diff
@@ -0,0 +1,302 @@
+import styled, { css, useTheme } from "styled-components";
+import { RadioButton } from "../../radio-button/radio-button";
+import { Fragment, useEffect } from "react";
+import { useAppDispatch, useAppSelector } from "../../../redux/hooks";
+import { ModalDialog } from "../../modal-dialog/modal-dialog";
+import {
+  selectArcGisContent,
+  selectArcGisContentSelected,
+  selectSortAscending,
+  selectSortColumn,
+  selectStatus,
+  setSortAscending,
+  setSortColumn,
+  setArcGisContentSelected,
+  resetArcGisContentSelected,
+} from "../../../redux/slices/arcgis-content-slice";
+import {
+  ArcGisContentColumnName,
+  ExpandState,
+  CollapseDirection,
+} from "../../../types";
+import { LoadingSpinner } from "../../loading-spinner/loading-spinner";
+import { ExpandIcon } from "../../expand-icon/expand-icon";
+
+const SpinnerContainer = styled.div<{ visible: boolean }>`
+  position: absolute;
+  left: calc(50% - 22px);
+  top: calc(50% - 22px);
+  opacity: ${({ visible }) => (visible ? 1 : 0)};
+`;
+
+const Table = styled.table`
+  width: 584px;
+  border-collapse: collapse;
+`;
+
+const TableHeader = styled.thead`
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 19px;
+  color: ${({ theme }) => theme.colors.secondaryFontColor};
+  overflow: hidden;
+`;
+
+const TableHeaderItem1 = styled.th`
+  width: 44px;
+  padding: 0;
+`;
+const TableHeaderItem2 = styled.th`
+  width: 343px;
+  padding: 0;
+`;
+const TableHeaderItem3 = styled.th`
+  width: 149px;
+  padding: 0;
+`;
```

Comment:
It might be 1 component with `width` property

Reply:
Done.